src/components/Message.tsx

```
@@ -178,52 +179,42 @@ const Message: FunctionComponent<MessageProps> = memo(({ content, isUser, files
 	);
 
+
+
 	return (
 		<div class={`message ${isUser ? 'message-user' : 'message-ai'} ${hasOnlyMedia ? 'media-only' : ''}`}>
+			{/* Agent handles welcome messages - no special logic needed */}
 			<div class="message-content">
-				{/* Render message content first */}
+				{/* Show content if available */}
 				{content && (
 					<div dangerouslySetInnerHTML={{ __html: marked(content) }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)